gnomad_constraint/utils/constraint.py

```diff
@@ -396,6 +406,183 @@ def apply_models(
     return ht.annotate(obs_exp=ht.observed_variants / ht.expected_variants)
 
 
+def calculate_mu_by_downsampling(
+    genome_ht: hl.Table,
+    context_ht: hl.Table,
+    recalculate_all_possible_summary: bool = True,
+    omit_methylation: bool = False,
+    count_singletons: bool = False,
+    keep_annotations: Tuple[str] = (
+        "context",
+        "ref",
+        "alt",
+        "methylation_level",
+    ),
+    ac_cutoff: int = 5,
+    downsampling_level: int = 1000,
+    total_mu: float = 1.2e-08,
+    pops: Tuple[str] = (),
+    min_cov: int = 15,
+    max_cov: int = 60,
+    gerp_lower_cutoff: float = -3.9885,
+    gerp_upper_cutoff: float = 2.6607,
```

Comment:
I think we need to add some of these parameters to the globals of the returned HT so we can keep track of anything that has different values from the defaults